modules/module-info.php

```diff
@@ -335,6 +335,24 @@ function jetpack_custom_css_more_info() { ?>
 }
 add_action( 'jetpack_module_more_info_custom-css', 'jetpack_custom_css_more_info' );
 
+
+/**
+ * Masterbar
+ */
+function jetpack_masterbar_more_link() {
+	echo 'https://jetpack.com/support/masterbar/';
+}
+add_action( 'jetpack_learn_more_button_masterbar', 'jetpack_masterbar_more_link' );
+
+function jetpack_masterbar_more_info() {
+	esc_html_e(
+		"Improves the experience for Jetpack authors when they've transferred their WordPress " .
+		"site from WordPress.com to a self-hosted plan by providing a similar experience to the " .
+		"front-end navigation on WordPress.com."
```

Comment:
Here's an idea for an improvement to this description:

`Quickly access your Stats, Notifications, Posts and more on WordPress.com. The Toolbar is displayed for any user on the site that is connected to WordPress.com.`